docs/deploy/providers/digitalocean.md

```diff
@@ -2,6 +2,31 @@
 
 **Preset:** `digitalocean` ([switch to this preset](/deploy/#changing-the-deployment-preset))
 
-::: warning Work in Progress
-Documentation for this section is not yet complete.
-:::
+Nitro supports deploying on the [Digital Ocean App Platform](https://docs.digitalocean.com/products/app-platform/) with minimal configuration.
+
+## Set up application
+
+1. Create a new Digital Ocean app following the [guide](https://docs.digitalocean.com/products/app-platform/how-to/create-apps/).
+
+1. Next, you'll need to configure environment variables. In your app settings, ensure the following app-level environment variables are set:
+
+   ```bash
+   HOST=0.0.0.0
```

Comment:
I believe we could include it in default node server

Reply:
https://github.com/unjs/nitro/commit/b7c719349a0ee399a525937abb5905e50b0351f2

Reply:
You mean, listen on public interface by default?